halo/attest/types/types.go

```diff
@@ -0,0 +1,4 @@
+package types
+
+// ChainNameFunc is a function that returns the name of a chain given its ID.
+type ChainNameFunc func(chainID uint64) string
```

Comment:
This should not be a type since we do not do anything with this type. A good example would be `http.HandlerFunc` - it is used as `http.Handler`. However, in this case we do not do anything with this function type. This is why I'd replace it with an interface - see comment above ^